### ISSUE TITLE
How to represent errors in query parameters?

### ISSUE THREAD

**Issue by emma-borhanian** [json-api/json-api#584]
If there is a problem a query parameter, is there any way to include structured data in the `errors` array describing which query parameter the problem was found in?

For example:

`GET /people?filter[namee]=smith`

``` json
{
  "errors":[{
    "status": "422",
    "code": "invalid_field",
    "title": "Invalid Field",
    "detail": "Invalid Field \"namee\"",
    "parameter": "filter[namee]"
  }]
}
```

**Comment by ethanresnick** [comment]
As the spec says, "Additional members **MAY** be specified within error objects.", so the JSON in your example is totally valid. That said, generic clients won't know what to do with the `"parameter"` value.

If you submitted a PR to make `"parameter"` part of the standard, as an analogue to the already-standard `"paths"` parameter, I'd support that.

**Comment by dgeb** [comment]
Closed via #638.

This can now be handled with the `parameter` member inside the `source` member of an error.
